Ch18/Ch18.md

```diff
@@ -286,14 +285,13 @@ individual database level.
 
 To clarify the terminology: a database in RavenDB can refer to a database group (a named database in the cluster, which resides
 on one or more nodes) or a database instance (a named databased on a particular node in the cluster). We don't usually have to
-distinguish between them because RavenDB let you work at the database group or database instance level transparently. 
+distinguish between them because RavenDB lets you work at the database group- or database instance-level transparently. 
```

Comment:
The '-' here looks strange

Reply:
So technically, the '-' is correct! This is because written out in its entirety, it would be "group-level or database-level." If you're not using both "level"s, you'd keep the hyphen. But if you feel it looks strange, you can cut it; it's not a hard-and-fast rule.

Reply:
okay